pdsClient/api.go

```diff
@@ -74,6 +74,11 @@ type WriteRecordRequest = Record
 // WriteRecordResponse represents a response from trying to write a record via the e3db api.
 type WriteRecordResponse = Record
 
+// DeleteRecordRequest represents a request to delete a record via e3db api.
+type DeleteRecordRequest struct {
+	RecordID string `json:"record_ids"`
```

Comment:
`json:"record_id"` vs ids?

Reply:
good catch, In the swagger spec it is actually `record_ids`, but I don't serialize it here so I'll switch